src/rawchannel.cc

```diff
@@ -19,6 +19,10 @@
  * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
  * IN THE SOFTWARE.
  */
+ 
+ /*
+  * 
+  */
```

Comment:
Empty comments?

Reply:
It's my first fork so I update the code for myself first and I put here some meta information not for the community.
I remove them